test/extended/image_ecosystem/s2i_perl.go

```diff
@@ -41,15 +42,19 @@ var _ = g.Describe("[image_ecosystem][perl][Slow] hot deploy for openshift perl
 			}
 		})
 
-		g.Describe("Dancer example", func() {
-			g.It(fmt.Sprintf("should work with hot deploy"), func() {
+		g.Describe("hot deploy test", func() {
+			g.It(fmt.Sprintf("should work"), func() {
```

Comment:
Shouldn't need the fmt.Sprintf as no formatting is happening, maybe there was at one time?

Reply:
yeah will update.